src/components/dropdownSearch/DropdownPrepend.tsx

```diff
@@ -0,0 +1,11 @@
+// tslint:disable-next-line:no-unused-variable
+import * as React from 'react';
+
+export interface IDropdownPrependProps {
+  prepend: JSX.Element | string;
+  key?: string;
+  children?: any;
+}
+
+export const DropdownPrepend = (props: IDropdownPrependProps): JSX.Element =>
```

Comment:
Can you use a `<Content/>` instead? It looks quite similar

Reply:
done